components/common/CommonDetail/CommonDetail.tsx

```diff
@@ -2,40 +2,51 @@
 
 import { ButtonStyle, ButtonSize } from "types/enums/button.enum";
 import { CommonLayout, CommonBtn } from "components/common";
+import { useRouter } from "next/navigation";
 import styles from "./CommonDetail.module.scss";
 
-type Detail = "EMPLOYER" | "EMPLOYEE";
+type Detail = "EMPLOYER" | "EMPLOYEE" | "APPLICATIONDETAILS";
```

Comment:
`APPLICATION_DETAILS`는 어떤가요
```suggestion
type Detail = "EMPLOYER" | "EMPLOYEE" | "APPLICATION_DETAILS";
```

Reply:
수정 완료